src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -238,6 +242,33 @@ export default class Mutations extends React.Component<
         }
     }
 
+    @autobind
+    @action
+    protected getDownloadData(dataType?: DataType): string {
+        var flatdata: any = '';
+        if (
+            this.selectedGene &&
+            this.props.store.getMutationMapperStore(this.selectedGene)
+        ) {
+            var mutationMapperStore: ResultsViewMutationMapperStore;
+            mutationMapperStore = this.props.store.getMutationMapperStore(
+                this.selectedGene
+            )!;
+            let mutations = mutationMapperStore.getMutationData();
+            let downloadData: any[] = [];
+            let sidToStudy = mutationMapperStore.studyIdToStudy.result!;
+            mutations.forEach(m => {
+                let study: any;
+                study = sidToStudy[m.studyId];
+                Object.assign(m, { StudyName: study.name });
```

Comment:
this is actually going to mutate the mutation data.  would be better to do 
```
Object.assign({},m,{...})
```
that will create new object with properties of both 2nd and 3rd args without mutating m

Reply:
Let's name the field `studyName`, not `StudyName`